.circleci/config.yml

```diff
@@ -128,6 +128,7 @@ workflows:
   steve:
     unless: << pipeline.parameters.turf-revision >>
     jobs:
+      - sanity-check
```

Comment:
Should this be a part of the release workflow(or maybe not the release workflow per se but part of release branch checks)? Since the license is taken from GL Native - it should stay the same as long as we are using the same GL Native version.

Reply:
Right but I think we are going to add more checks to this job that make sense on each build run

Reply:
It just if I understand correctly - this check fails when contents of GL Native's `LICENSE-iOS.md` differs from `LICENSE.md` and this can happen regardless of the contents of the PR, it would be quite confusing when this happens. 
I wonder if it is possible to run this job only when dependencies are changed - i.e. when Package.swift and/or MapboxMaps.podspec is changed.

Reply:
It is already related to the dependency update PR in implicit way. We download `LICENSE-iOS.md` from the pinned GL Native version and not from the `main` branch. If we assume that GL Native team wouldn't recreate tag with different license file content it would never fail for a usual PRs

Reply:
Yeah, using a pinned version would do the trick 😀

Reply:
Fixed! 🥳